src/core/ruby_version.rs

```diff
@@ -485,13 +493,13 @@ macro_rules! get_stack_trace(
         fn stack_base(thread: &$thread_type) -> i64 {
             // Ruby stack grows down, starting at
             //   ruby_current_thread->stack + ruby_current_thread->stack_size - 1 * sizeof(rb_control_frame_t)
-            // I don't know what the -1 is about. Also note that the stack_size is *not* in bytes! stack is a
-            // VALUE*, and so stack_size is in units of sizeof(VALUE).
+            // We must skip two dummy frames:
+            // https://github.com/ruby/ruby/blob/5445e0435260b449decf2ac16f9d09bae3cafe72/vm_backtrace.c#L477-L485%C2%A
```

Comment:
Note that i think the other -1 is occurring implicitly in our calculation of how many CFPs to collect, hence why this is -1 and not -2 here.